tests/spec/jasmine-event-recur.js

```diff
@@ -207,6 +207,13 @@ describe("The Calendar Interval", function() {
         expect(recurrence.matches( moment(startDate).date(27) )).toBe(false);
     });
 
+    it("weeksOfMonthByDay should work", function() {
```

Comment:
Can you take a quick look at this test in context? https://github.com/rmarscher/moment-recur/blob/nth-weekday-of-month/tests/spec/jasmine-event-recur.js#L210  Should I pull the new tests out into a separate feature marked by `describe` or leave where they are? Thanks.